_posts/2013-08-07-employees.markdown

```diff
@@ -19,7 +19,7 @@ title: Employees
 | `company_id`                  | Integer           |     X     |          |         | employee's company ID
 | `manager_id`                  | Integer           |     X     |          | null    | employee's manager ID
 | `department`                  | String            |     X     |          | null    | employee's department.
-| `ssn`                         | String            |           |    X     | null    | masked social security number. When updating this attribute, format should be a string of nine numbers (no dashes.)
+| `ssn`                         | String            |           |    X     | null    | Updates only (reads have been deprecated - this returns an empty string since July 2020). When updating this attribute, format should be a string of nine numbers (no dashes.)
```

Comment:
Should the default column say empty string instead of null here now?

Reply:
Ah yes I can change that! :)